src/main/java/controllers/ControllerConstants.java

```diff
@@ -0,0 +1,20 @@
+package controllers;
```

Comment:
Comments from 
https://github.com/Raman-Piachko/stateful-expression-calculator/pull/4
valid for this PR